packages/braintree-integration/src/is-us-bank-account-instrument-like.ts

```diff
@@ -0,0 +1,18 @@
+import {
+    PaymentInstrument,
+    UsBankAccountInstrument,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+export default function isUsBankAccountInstrumentLike(
+    instrument?: PaymentInstrument,
+): instrument is UsBankAccountInstrument {
+    if (!instrument) {
+        return false;
+    }
+
+    return (
+        'accountNumber' in instrument &&
+        'routingNumber' in instrument &&
+        'ownershipType' in instrument
+    );
+}
```

Comment:
Should we check all required fields what presents in UsBankAccountInstrument to create the typeguard?

Reply:
I thought that it was enough to check a few required fields. But check all will not be superfluous.